scripts/globals/combat/physical_utilities.lua

```diff
@@ -107,45 +107,61 @@ local elementalBelt = -- Ordered by element.
 -- 'fSTR' in English Wikis. 'SV function' in JP wiki and Studio Gobli.
 -- BG wiki: https://www.bg-wiki.com/ffxi/FSTR
 -- Gobli Wiki: https://w-atwiki-jp.translate.goog/studiogobli/pages/14.html?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp
+-- Mob calculation: https://docs.google.com/spreadsheets/d/1YBoveP-weMdidrirY-vPDzHyxbEI2ryECINlfCnFkLI/edit?gid=224123492#gid=224123492&range=C50
 xi.combat.physical.calculateMeleeStatFactor = function(actor, target)
     local fSTR = 0 -- The variable we want to calculate.
 
     -- Calculate statDiff.
     local statDiff     = actor:getStat(xi.mod.STR) - target:getStat(xi.mod.VIT)
-    local weaponRank   = actor:getWeaponDmgRank()
-    local statLowerCap = (7 + weaponRank * 2) * -2
-    local statUpperCap = (14 + weaponRank * 2) * 2
-
-    statDiff = utils.clamp(statDiff, statLowerCap, statUpperCap)
-
-    -- Calculate fSTR based on stat difference.
-    if statDiff >= 12 then
-        fSTR = statDiff + 4
-    elseif statDiff >= 6 then
-        fSTR = statDiff + 6
-    elseif statDiff >= 1 then
-        fSTR = statDiff + 7
-    elseif statDiff >= -2 then
-        fSTR = statDiff + 8
-    elseif statDiff >= -7 then
-        fSTR = statDiff + 9
-    elseif statDiff >= -15 then
-        fSTR = statDiff + 10
-    elseif statDiff >= -21 then
-        fSTR = statDiff + 12
+
+    -- Pets and mobs have own calculation
+    if actor:isMob() or actor:isPet() then
+        local fSTRupperCap = 24
+        local fSTRlowerCap = -20
+        fSTR = math.floor((statDiff + 4) / 4)
+        fSTR = utils.clamp(fSTR, fSTRlowerCap, fSTRupperCap)
+        -- level 1 mobs always have fSTR of 1
+        if actor:isMob() and actor:getMainLvl() == 1 then
```

Comment:
I think this should be `<= 1`, at some point we may actually support level -1 in some meaningful way, but level 0 is technically "valid" too. I remember helping to confirm this fSTR of 1 though, that was a trip

Reply:
Now would be a good time to ask what calculation trusts use

Reply:
> I think this should be `<= 1`, at some point we may actually support level -1 in some meaningful way, but level 0 is technically "valid" too. I remember helping to confirm this fSTR of 1 though, that was a trip

Yep, thanks for info, I have updated the PR!